### PR TITLE
My wallet back

### PR DESCRIPTION
**Pull request by engshorouq**
Relates #36 
## Description
 - make the backend of the my wallet (contains pre-tour and my wallet page)
 - make tests for routers and queries
## Fix bugs
 - create new fake database to make testing

### CODE REVIEW

**Review by IsraaSulaiman — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "NODE_ENV=test tape test/testQueries.js | tap-spec & tape test/testRoutes.js | tap-spec",
+    "test": "NODE_ENV=test tape test/testQueries.js | tap-spec",
+    "testRoutes": "NODE_ENV=test tape test/testRoutes.js | tap-spec",
```
Comment (IsraaSulaiman) [inline package.json]: Travis only reads and executes test configuration. It won't execute the command for 'testRoutes'. That's why I made both of them within 'test' config.

I did check Travis and it only says: 

Screenshot from 2019-03-21 19-41-16

Comment (engshorouq) [reply package.json]: i solve it
but the script line must be like that 
```
"test": "NODE_ENV=test tape test/testQueries.js | tap-spec && NODE_ENV=test tape test/testRoutes.js | tap-spec",
```

File: src/database/dbBuild.sql [hunk 2]
```diff
@@ -42,6 +42,37 @@ BEGIN;
     date TEXT NOT NULL,
     description TEXT
   );
-
+  INSERT INTO category
```
Comment (asmaa-ezz) [inline src/database/dbBuild.sql:45]: in file `dbBuild.sql` just content the schema 
you can create another file to name **fackData.sql**
to write the query

**Review by KhaderMurtaja — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (IsraaSulaiman) [reply package.json]: :+1:

File: src/database/dbBuild.sql (hunk 2 above)
Comment (IsraaSulaiman) [reply src/database/dbBuild.sql:45]: @asmaa-ezz what about the categories? They are static and I think they should be in the SQL.

**Pull request merged**